features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/dashdirect/DashDirectViewModel.kt

```diff
@@ -91,19 +91,14 @@ constructor(
     val usdExchangeRate: LiveData<ExchangeRate>
         get() = _exchangeRate
 
-    var purchaseGiftCardDataMerchant: Merchant? = null
-    var purchaseGiftCardDataPaymentValue: Pair<Coin, Fiat>? = null
+    lateinit var giftCardMerchant: Merchant
+    lateinit var giftCardPaymentValue: Fiat
```

Comment:
We can assume that these values are initialized by the time they are needed. This allows us to avoid all the dances will null-checks.
If they aren't, we have a problem somewhere.